sql/updates/latest-dev.sql

```diff
@@ -41,8 +41,8 @@ CREATE AGGREGATE _timescaledb_internal.finalize_agg(agg_name TEXT,  inner_agg_co
     FINALFUNC_EXTRA
 );
 
-ALTER TABLE _timescaledb_catalog.installation_metadata RENAME TO telemetry_metadata;
-ALTER INDEX _timescaledb_catalog.installation_metadata_pkey RENAME TO telemetry_metadata_pkey;
+ALTER TABLE _timescaledb_catalog.installation_metadata RENAME TO metadata;
```

Comment:
I would just keep the old name `installation_metadata` and remove these lines from the update script.

Reply:
Unfortunately the `installation_metadata` to `telemetry_metadata` rename is going out in 1.3 which is already frozen.  So I'll have to change this to rename from `telemetry_metadata` to whatever we want the new name to be.  

As far as whether we want to go with `installation_metadata` again, I actually found that name to be confusing as I was reading as metadata needed or created at install time (I realize now that was my parsing error).  I still prefer simply `metadata` as a generic catch-all table name, but if you have a strong preference we can have a broader discussion.

Reply:
I would prefer to keep the telemetry metadata separate from metadata required for operation.

Also the 1.3.0 release PR will empty this file so this change has to be added after 1.3.0 has been released